src/Spice86.Core/Emulator/InterruptHandlers/Common/MemoryWriter/MemoryAsmWriter.cs

```diff
@@ -95,7 +95,7 @@ public void WriteFarCallToSwitcherDefaultAddress(InMemoryAddressSwitcher inMemor
     public SegmentedAddress WriteFarCall(SegmentedAddress destination) {
         WriteUInt8(0x9A);
         // Make a copy of the address since it is going to be modified by our writes.
```

Comment:
Comment can be removed

Reply:
I thought so too at first. But it does explain why we should not move this to `return CurrentAddress`

Reply:
Indeed, my bad